.gitattributes

```diff
@@ -1 +1,13 @@
 * text=auto eol=lf
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/config_sentence_transformers.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/modules.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/special_tokens_map.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/tokenizer.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/config.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/2_Normalize filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/README.md filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/pytorch_model.bin filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/sentence_bert_config.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/tokenizer_config.json filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/vocab.txt filter=lfs diff=lfs merge=lfs -text
+tests/sentence_transformers/test_resources/test_model/model.sentence_transformer/1_Pooling filter=lfs diff=lfs merge=lfs -text
```

Comment:
Can we use a smalle model instead of sentence transformers, e.g. scikit-learn, so that we don't need to use git LFS to host the test files?